sentry-delayed_job/lib/sentry/delayed_job/plugin.rb

```diff
@@ -40,8 +49,15 @@ def self.generate_contexts(job)
           queue: job.queue,
           created_at: job.created_at,
           last_error: job.last_error&.byteslice(0..1000),
-          handler: job.handler&.byteslice(0..1000)
+          handler: job.handler&.byteslice(0..1000),
+          handler_class: job.payload_object.class,
```

Comment:
is it necessary to have both `handler_class` and `job_class`? I think the latter should include the former?

Reply:
No, it is not. handler_class was the first version and I forgot to delete.